### PR TITLE
Webpack & scss

### PR DESCRIPTION
**Pull request by deathbearbrown**
I have to do a ton of CSS work and I need css watch, so I updated webpack to handle scss files.
By putting the scss stuff in webpack, we are now making one less request in the app. 

In this PR I have the updated webpack stuff & I have moved the JSX files about for my scaffold restructure.  I wanted to get this mini PR in first with these changes. 

To test this run `npm start` and make sure it's doing it's thing. The only difference you should see if a checkered BG which is my test that images are working when referenced in the scss.

### CODE REVIEW

File: src/js/components/Sidebars.jsx [hunk 1]
```diff
@@ -1,11 +1,11 @@
...
-    PipelineList = require('./pipelines/PipelineList'),
-    ScaleList = require('./ScaleList'),
-    LayerList = require('./LayerList'),
-    Inspector = require('./Inspector'),
+    InspectorSidebar = require('./InspectorSidebar'),
+    VisualSidebar = require('./VisualSidebar'),
+    PipelinesSidebar = require('./PipelinesSidebar'),
     model = require('../model');
 
+// Splitting each sidebar into it's column
```
Comment (kadamwhite) [inline src/js/components/Sidebars.jsx]: nit: `its`

File: webpack.config.js [hunk 2]
```diff
@@ -1,5 +1,8 @@
+/* eslint-disable */
```
Comment (kadamwhite) [inline webpack.config.js:1]: Why is this necessary?

Comment (deathbearbrown) [inline webpack.config.js:1]: because eslint was going crazy when I ran 'lint:branch'

**Comment by kadamwhite** [comment]
We'll want to adapt this so that we can extract the CSS independently from the JS bundles at some point, but I'm +1 on what's here

File: webpack.config.js (hunk 2 above)
Comment (kadamwhite) [inline webpack.config.js:1]: Huh, odd. Ok.
